### ISSUE TITLE
Option to ad our own URL to stream radio.

### ISSUE THREAD

**Issue by mikhoul** [segler-alex/RadioDroid#532]
I know that we can submit new radio stations to  www.radio-browser.info but it would be useful also to be able to ad our own stations like https://MyStation.net/stream.mp3 .

Also when we add new "personal" URL maybe having an option to easily submit the link to  www.radio-browser.info from the app without having to register would be useful too.

Regards :octocat:

**Comment by werman** [comment]
Yeah, adding custom stations would be could, actually I think it shouldn't be hard to do.

**Comment by Whovian9369** [comment]
Sorry to poke over 1yr old issues, but I feel like this is one enhancement that would be VERY handy, especially for stations that I want to quickly connect to again, should an error occur.

**Comment by werman** [comment]
> Sorry to poke over 1yr old issues, but I feel like this is one enhancement that would be VERY handy, especially for stations that I want to quickly connect to again, should an error occur.
> Even subscribed to the issue, so I honestly hope something is to come from this!

It's ok, I'll try to add it in some trivial way, I think I'll just make a dialog to add custom station to favourites,
